docs: explain monetization change flow and remove rental pricing model as an option (#2333)
apify/apify-docs: sources/platform/actors/publishing/monetize/index.mdx

Reviewer: We should remove mentions of rental from descriptions as well

Maybe something akin to:
```suggestion
description: Monetize your Actors on Apify Store with pay-per-event pricing. Learn how to set up, configure, and manage monetization changes for your published Actors.
```

#### Significant changes

The following changes are considered significant and require a _14-day notice period_:

- Changing the pricing model (e.g., from rental to pay-per-event)
- Increasing prices
- Adding new paid events

When you submit a significant change, users of your Actor are notified and the change is scheduled to take effect after 14 days. During this waiting period, the current pricing remains active.

#### Non-significant changes

The following changes take effect **immediately**:

- Decreasing prices
- Removing events
- Updating event descriptions
- Adjusting other non-pricing settings

#### Restrictions on significant changes

:::caution You cannot cancel a planned change

Once you commit to a significant change, you cannot cancel or modify it yourself. If you need to revert a planned change, contact [Apify support](https://apify.com/contact).

:::

Significant changes are limited to _once per month_ per Actor. After submitting a significant change, you must wait for the full cycle to complete before making another one:

> 1. You submit a significant change (e.g., a price increase).

Reviewer 2: ```suggestion
1. You submit a significant change. For example, a price increase.
```

You can change the monetization setting of your Actor by using the same wizard as for the setup in the **Monetization** section of your Actor's **Publication** tab.

Changes are split into two categories: _significant_ and _non-significant_. Non-significant changes take effect immediately, while significant changes require a 14-day notice period to give your users time to adjust.

#### Significant changes

> The following changes are considered significant and require a **14-day notice period**:

Reviewer: Bold is for UI elements only.

```suggestion
The following changes are considered significant and require a _14-day notice period_:
```

> - Changing the pricing model (e.g. from rental to pay-per-event)

Reviewer: ```suggestion
- Changing the pricing model (e.g., from rental to pay-per-event)
```

- Increasing prices
- Adding new paid events

When you submit a significant change, users of your Actor are notified and the change is scheduled to take effect after 14 days. During this waiting period, the current pricing remains active.

#### Non-significant changes

The following changes take effect **immediately**:

- Decreasing prices
- Removing events
- Updating event descriptions
- Adjusting other non-pricing settings

#### Restrictions on significant changes

:::caution You cannot cancel a planned change

Once you commit to a significant change, you cannot cancel or modify it yourself. If you need to revert a planned change, contact [Apify support](https://apify.com/contact).

:::

> Significant changes are limited to **once per month** per Actor. After submitting a significant change, you must wait for the full cycle to complete before making another one:

Reviewer: Bold is for UI elements only

```suggestion
Significant changes are limited to _once per month_ per Actor. After submitting a significant change, you must wait for the full cycle to complete before making another one:
```

> 1. You submit a significant change (e.g. a price increase).

Reviewer: ```suggestion
1. You submit a significant change (e.g., a price increase).
```

1. The change takes effect after **14 days**.
1. You can then submit your next significant change, which also takes **14 days** to apply.

This means approximately **one month** passes between committing to one significant change and having the next one take effect. Plan your pricing strategy carefully before committing.

> 

Reviewer: Bold is for UI elements only

```suggestion
1. The change takes effect after _14 days_.
1. You can then submit your next significant change, which also takes _14 days_ to apply.

This means approximately _one month_ passes between committing to one significant change and having the next one take effect. Plan your pricing strategy carefully before committing.

```
